library/psa_crypto_core.h

```diff
@@ -30,6 +30,26 @@
 #include "psa/crypto.h"
 #include "psa/crypto_se_driver.h"
 
+/** Constant-time buffer comparison
+ *
+ * \param[in]  a    Left-hand buffer for comparison.
+ * \param[in]  b    Right-hand buffer for comparison.
+ * \param n         Amount of bytes to compare.
+ *
+ * \return 0 if the buffer contents are equal, non-zero otherwise
+ */
+static inline int mbedtls_psa_safer_memcmp(
```

Comment:
I think the only reason is that currently in development and in psa_crypto,c this is a `static inline` function.

Reply:
I guess it started out as `static inline` in `ssl_internal.h` so that it could be shared by client and server code, and we copied it several times as `static inline` without paying attention. Ok, it doesn't matter, we'll resolve this when we create the constant-time module.